experimental/devices/cap1188/cap1188.go

```diff
@@ -0,0 +1,541 @@
+		// are more sensitive to noise, however, and a noisy environment may flag more false touches with higher
+		// sensitivity levels.
+		// Set to 4x
+		// TODO(mattetti): make that configurable.
+		byte(1)<<6 | byte(0)<<5 | byte(1)<<4 |
+		byte(0)<<3 | byte(0)<<2 | byte(0)<<1 | byte(0)<<0)
+	if d.Debug {
+		log.Printf("cap1188: Sensitivity mask: %08b\n", sensitivity)
+	}
+	if err = d.regWrapper.WriteUint8(0x1F, sensitivity); err != nil {
+		return fmt.Errorf("failed to set sensitivity - %s", err)
+	}
+
+	if opts.LinkedLEDs {
+		if err = d.LinkLEDs(); err != nil {
+			return err
+		}
+	}
+
+	if opts.RetriggerOnHold {
+		if err = d.regWrapper.WriteUint8(0x28, 0xff); err != nil {
+			return fmt.Errorf("failed to set retrigger on hold - %s", err)
+		}
+	} else {
+		if err = d.regWrapper.WriteUint8(0x28, 0x00); err != nil {
+			return fmt.Errorf("failed to turn off retrigger on hold - %s", err)
+		}
+	}
+
+	// page 47 - configuration registers
+	config := (
+	// Timeout: Enables the timeout and idle functionality of the SMBus protocol.
+	// default 0: The SMBus timeout and idle functionality are disabled. The
+	// SMBus interface will not time out if the clock line is held low.
+	// Likewise, it will not reset if both data and clock lines are held
+	// high for longer than 200us
+	byte(0)<<7 |
+		// Configures the operation of the WAKE pin.
+		// default 0: The WAKE pin is not asserted when a touch is detected while the
+		// device is in Standby. It will still be used to wake the device from
+		// Deep Sleep when driven high.
+		byte(0)<<6 |
+		// digital noise threshold
+		// Determines whether the digital noise threshold is used by the device.
+		// default 1:  The noise threshold is disabled. Any delta count that
+		// is less than the touch threshold is used for the automatic
+		// re-calibration routine.
+		byte(1)<<5 |
+		// analog noise filter
+		// default 0: Determines whether the analog noise filter is enabled. Setting this
+		// bit disables the feature.
+		byte(1)<<4 |
+		// maximum duration recalibration
+		// Determines whether the maximum duration recalibration is enabled.
+		//
+		// if 0, the maximum duration recalibration functionality is
+		// disabled. A touch may be held indefinitely and no re-calibration
+		// will be performed on any sensor input.
+		//
+		// if 1, The maximum duration recalibration functionality is
+		// enabled. If a touch is held for longer than the
+		// opts.MaxTouchDuration, then the re-calibration routine will be
+		// restarted
+		recalFlag<<3 |
+		byte(0)<<2 |
+		byte(0)<<1 |
+		byte(0)<<0)
+	if d.Debug {
+		log.Printf("cap1188: Config mask: %08b\n", config)
+	}
+	if err = d.regWrapper.WriteUint8(0x20, config); err != nil {
+		return fmt.Errorf("failed to set the device configuration - %s", err)
+	}
+
+	config2 := (
+	// Linked LED Transition controls
+	// default 0: The Linked LED Transition controls set the min duty
+	// cycle equal to the max duty cycle
+	byte(0)<<7 |
```

Comment:
You can document the bits you don't set but no need to add code for these (even if it's elided by the compiler). I generally do not document the functionality that are not expected to be used.